packages/tools/locales/en_US.po

```diff
@@ -4938,6 +4938,10 @@ msgstr ""
 msgid "Toggle external editing"
 msgstr ""
 
+#: packages/app-desktop/gui/MainScreen/commands/toggleMenuBar.ts:9
+msgid "Toggle menu bar"
+msgstr ""
+
```

Comment:
You don't need to add this here - that will be done automatically as long as you wrap the translatable strings in `_(...)` which you did

Reply:
I've removed this in the latest commit